serpentTools/tests/refrect_fmtx0.m

```diff
@@ -0,0 +1,70 @@
+
+% ----- Analog fission matrixes
+
+fmtx_xmin = -6.30000E-01 ;
+fmtx_xmax = 6.30000E-01 ;
+fmtx_nx = 1 ;
+fmtx_ymin = -6.30000E-01 ;
+fmtx_ymax = 6.30000E-01 ;
+fmtx_ny = 1 ;
+fmtx_zmin = 0.00000E+00 ;
+fmtx_zmax = 3.66000E+02 ;
+fmtx_nz = 10 ;
+
+fmtx_t = zeros(11,10);
```

Comment:
Delete the file after this line.
The test method inspects if this raises an error at this specific point, and all that remains after this file is not necessary, if the reader is functioning properly